package.json

```diff
@@ -75,7 +75,25 @@
     "webdriver-manager": "^12.1.4",
     "webpack": "^4.30.0",
     "webpack-cli": "^3.3.1",
-    "yargs": "^13.2.2"
+    "yargs": "^13.2.2",
+    "@angular/common": "^8.0.0",
+    "@angular/compiler": "^8.0.0",
+    "@angular/core": "^8.0.0",
+    "@angular/forms": "^8.0.0",
+    "@angular/platform-browser": "^8.0.0",
+    "@angular/platform-browser-dynamic": "^8.0.0",
+    "@angular/router": "^8.0.0",
+    "@angular/upgrade": "^8.0.0",
+    "angular": "1.6.6",
+    "angular-route": "1.6.6",
+    "bootstrap": "3.3.7",
+    "core-js": "^2.5.3",
+    "jquery": "^2.2.4",
+    "lodash": "4.17.4",
+    "moment": "~2.17.1",
+    "reflect-metadata": "^0.1.12",
+    "rxjs": "^6.4.0",
+    "zone.js": "^0.9.1"
```

Comment:
Order these alphabetically with the other ones too.

Reply:
Done